### PR TITLE
Improving-doc

### PR DESCRIPTION
**Pull request by ducng0610**
- Update README
- Not sure why in develop branch the vagrant_config_override.example.json was removed so I readd it, since it is still be mentioned in README

### CODE REVIEW

**Comment by hoatle** [comment]
@duc4nh that's new feature from teracy-dev v0.5.0-c1, we're going to use https://github.com/acme101/dev-setup/blob/develop/vagrant_config_default.json instead.

File: README.md [hunk 1]
```diff
@@ -57,18 +57,18 @@ environment for `Acme` organization, so we need to create `acme-dev` directory.
...
-    $ git clone git@github.com:acme101/dev-setup.git
+    $ git clone https://github.com/acme101/dev-setup.git
     ```
 
 - Type the following command:
-    
+
     ```bash
     $ cd ~/acme-dev
     $ cp workspace/dev-setup/vagrant_config_override.example.json vagrant_config_override.json
```
Comment (hoatle) [inline README.md]: remove this

File: vagrant_config_override.example.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (hoatle) [inline vagrant_config_override.example.json]: remove this file

**Review by hoatle — CHANGES_REQUESTED** [review]

**Comment by ducng0610** [comment]
Ok, let me update the readme instead. It still mention `cp workspace/dev-setup/vagrant_config_override.example.json vagrant_config_override.json`

**Comment by ducng0610** [comment]
does this file vagrant_config_default.json work just by putting it at teracy-dev root? or a rename needed?

**Comment by hoatle** [comment]
workspace/dev-setup/vagrant_config_default.json is auto discovered by teracy-dev, users don't have to do anything.

For each vagrant_config_default.json, users can create vagrant_config_override.json files themselves when they want to custom anything for their usage. However, in most cases, default configs are enough.

**Comment by hoatle** [comment]
@duc4nh see more: https://github.com/teracyhq/dev/issues/315

**Comment by ducng0610** [comment]
requested changes applied

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**